src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(foo, number) {
```

Comment:
I would place operate under other operations to be consistent, but it's not a big deal)
Name `foo` is not really descriptive, consider renaming `foo` to `callback` or `operation`